### DISCUSSION TITLE
How to remove separator line when using background left or right image

### DISCUSSION THREAD

**Discussion by SoumayaMauthoorMOJ** [marp-team/marp#537] [Q&A] [answered]
When I use the `bg right` directive marp introduces a line separating the two sections for example [here](https://moj-analytical-services.github.io/dmet-cfe/#2) . How can I remove this line?

The markdown is defined [here](https://github.com/moj-analytical-services/dmet-cfe/edit/main/mission_statement/README.md) on line 21

**Comment by yhatt** [comment]
I cannot see such a line. Where is the line are you mentioned?

Attaching a screenshot may have a better chance that Marp community can resolve the issue.

**Comment by SoumayaMauthoorMOJ** [comment]
Hi @yhatt apologies here is the screenshot:

![image](https://github.com/user-attachments/assets/1f2ed297-ba1b-42da-91ae-4fd30ea7acf0)

There is a blue line between the two sections

The blue line disappears if I use `bg left`

**Reply by yhatt** [reply, marked as answer]
Similar: https://stackoverflow.com/questions/51681584/issue-with-inset-box-shadow-and-transform-scale-on-chrome

`box-shadow` with the `inset` keyword creates a filled area around the slide container. It seems to be leaking the color from the around if the scaling for the section element was specific scale.

One idea to avoid leaks the color is to make the fill area a little thinner by using 4th parameter of `box-shadow` property `<spread-radius>`.

```diff
-box-shadow: inset 0 -60px 0 var(--primary-color);
+box-shadow: inset 0 -61px 0 -1px var(--primary-color);
```

**Reply by SoumayaMauthoorMOJ** [reply]
Perfect!
